### PR TITLE
unit tests for ag_edit_participants, initial commit.

### PR DESCRIPTION
**Pull request by sjanssen2**

### CODE REVIEW

File: knimin/tests/test_ag_edit_participant.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            if key == 'zip':
+                key = 'zipcode'
+            elif key == 'ag_login_id':
+                continue
+            self.assertIn(('</td><td><input type="text" name="%s" id="%s" '
+                           'value="%s"></td></tr>') % (key, key, value),
+                          response.body)
+
+        # check what happens if user with email does not exist.
+        # TODO: we should create a better error message in the handler to be
```
Comment (wasade) [inline knimin/tests/test_ag_edit_participant.py]: agree, could you create an issue please?

File: knimin/tests/test_ag_edit_participant.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                              'country': country,
+                              'ag_login_id': ag_login_id})
+        self.assertEqual(response.code, 400)
+        self.assertIn(('MissingArgumentError: HTTP 400: Bad Request '
+                       '(Missing argument address)'), response.body)
+
+
+if __name__ == "__main__":
+    main()
+# http://localhost:8889/ag_edit_participant/?email=xjztuew@wbfznvoyxb.com
```
Comment (wasade) [inline knimin/tests/test_ag_edit_participant.py]: ?

**Comment by wasade** [comment]
Just minor comments, thanks @sjanssen2!

**Comment by wasade** [comment]
:+1:

**Comment by mortonjt** [comment]
👍

**Pull request merged**